test/e2e/network/network_policy.go

```diff
@@ -277,6 +277,38 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			testCanConnect(f, f.Namespace, "client-a", service, 80)
 			testCanConnect(f, f.Namespace, "client-b", service, 81)
 		})
+		It("should allow ingress access on one named port", func() {
```

Comment:
@bcreane you need to add `[Feature:NetworkPolicy]` to this test

Reply:
understood, done.